python/test/gherkin_in_markdown_token_matcher_test.py

```diff
@@ -8,7 +8,7 @@
 
 def test_it_matches_FeatureLine():
     tm = GherkinInMarkdownTokenMatcher('en')
-    line = GherkinLine('''## Feature: hello''',location['line'])
```

Comment:
Maybe I'm missing something here?  I'm not sure this test has this change?

Reply:
nice catch! not sure how this happened... Fixed in abed0db7f255cb95deb92699c4d5590abddafe22